server/routes/getReqs/getBoard.ts

```diff
@@ -0,0 +1,9 @@
+import { Router } from "express";
```

Comment:
Can you revert all the server side changes? Or put them on another branch?

Reply:
Should I just delete the server folder? Or should we merge this in for now & make a new refactor branch to change up the folder structure and delete the server stuff since we arent gonna use it @KevinLu9 @Eeyore6

Reply:
I'm fine with the deletion of the server folder. @KevinLu9 thoughts?

Reply:
im happy for you to delete the server folder.